releases/latest/postgresql-bundle.yaml

```diff
@@ -22,7 +20,9 @@ machines:
     constraints: arch=amd64
   '1':
     constraints: arch=amd64
+name: postgresql-bundle
 relations:
 - - pgbouncer:backend-database
   - postgresql:database
 series: focal
+type: bundle
```

Comment:
It seems that this hasn't worked correctly as it only changed the order of the fields in the bundle file. But the reason is because [yaml.dump](https://github.com/canonical/postgresql-bundle/blob/53be3d43d4202822a082f4a818aae3db9cbd31fe/update_bundle.py#L34) by default sort the dict keys (there is a sort_keys parameter that can be set to False). So it's fine to merge this, as after that new PRs will not be created unless the revision changes (that's what we need).